river/Vector.zig

```diff
@@ -0,0 +1,62 @@
+// This file is part of river, a dynamic tiling wayland compositor.
+//
+// Copyright 2023 The River Developers
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+const std = @import("std");
+const math = std.math;
+const wlr = @import("wlroots");
+
+const Vector = @This();
+
+x: i32,
+y: i32,
+
+pub fn positionOfWlrBox(box: wlr.Box) Vector {
```

Comment:
We no longer use non-wlr boxes.
```suggestion
pub fn positionOfBox(box: wlr.Box) Vector {
```